### DISCUSSION TITLE
POSSIBLE TO USE SIGNAL W/O A PHONE # ?

### DISCUSSION THREAD

**Discussion by NoahChomskie** [AsamK/signal-cli#1197] [General]
My apologies if I happen to be barking up the wrong tree. 
I have a smart phone, but with $ being tight & projected to be even worse, I am not connected to a carrier. 
It is handy around the home using WiFi. 
I would like to add Signals for messaging. 
There are a number of sites that say this is possible wothout a phone/ number.
But as I have learned from experience, I could open 25 tabs, spend 3 hours, & end up even more confused.
In order to avoid my past behavior, I stopped & thought for a moment, that temporary haly brought me here. 
Is it POSSIBLE ?

**Comment by thlkmoeller** [comment]
Hello,

such a pity no one answered till now. The answer is a very clear yes. I am doing it myself, with even 3 different laptops. I don't own a smartphone. You need but a mobile number to register a signal account.

Please read this:

https://gist.github.com/szepeviktor/2c6a19cb91c4bb561369707f22bcf413

**Reply by NoahChomskie** [reply]
I appreciate you taking the time to respond to my dilemma but I lack the skills to implement the instructions you provided.
Thank you once again.

**Reply by thlkmoeller** [reply]
Don't underestimate Yourself :-) Actually it is a little tricky to do it. But I am also not an informatics guy. And I figured it out. You also just have to do it once. Then it works like a charm and You can use Signal with Your computer. I also do video calls and voice messages and so on. I never owned a smartphone. At which step are You stuck then?
